test/e2e/framework/volume/BUILD

```diff
@@ -0,0 +1,33 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["volume.go"],
+    importpath = "k8s.io/kubernetes/test/e2e/framework/volume",
```

Comment:
Will you create an OWNERS file for this dir with the same sig-storage people in it: https://github.com/kubernetes/kubernetes/blob/91567ae06c52ffa114fe9231c888efdda0e5c731/test/e2e/storage/OWNERS

Reply:
Thanks @fejta 
I'll be glad to do it.